--- a/src/freedreno/common/freedreno_devices.py	2026-07-02 18:10:47.417346600 +0500
+++ b/src/freedreno/common/freedreno_devices.py	2026-07-02 18:10:47.417346600 +0500
@@ -844,7 +844,7 @@
         gmem_vpc_attr_buf_size = 0xc000,
         ubwc_unorm_snorm_int_compatible = True,
         supports_uav_ubwc = True,
-        has_generic_clear = True,
+        has_generic_clear = False,
         r8g8_faulty_fast_clear_quirk = True,
         gs_vpc_adjacency_quirk = True,
         ubwc_all_formats_compatible = True,
