--- a/src/freedreno/common/freedreno_devices.py	2026-07-02 18:10:47.455171100 +0500
+++ b/src/freedreno/common/freedreno_devices.py	2026-07-02 18:10:47.455171100 +0500
@@ -839,7 +839,7 @@
         has_event_write_sample_count = True,
         load_inline_uniforms_via_preamble_ldgk = True,
         load_shader_consts_via_preamble = True,
-        has_gmem_vpc_attr_buf = True,
+        has_gmem_vpc_attr_buf = False,
         sysmem_vpc_attr_buf_size = 0x20000,
         gmem_vpc_attr_buf_size = 0xc000,
         ubwc_unorm_snorm_int_compatible = True,
